### PR TITLE
ADD Mikaila Shanin Sotaso

### PR DESCRIPTION
**Pull request by miksdumb**

### CODE REVIEW

File: js/modules/mikailashanin.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
+export default {
+    state: {
+        profile: {
+            username: "MikailaShaninSotaso",
```
Comment (kulotsystems) [inline js/modules/mikailashanin.js:4]: Please update your GitHub username.

```suggestion
            username: "miksdumb",
```

**Review by kulotsystems — CHANGES_REQUESTED** [review]

**Review by miksdumb — COMMENTED** [review]
Sir, can u check it out again please.

File: js/store.js [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 // import your .js file here (e.g. import john from './modules/john.js')
 // ...
+import miksdumb from './modules/miksdumb.js';
```
Comment (kulotsystems) [inline js/store.js:6]: You only changed your GitHub username but you did not rename your module. So, you can keep your previous module file name.

```suggestion
import miksdumb from './modules/mikailashanin.js';
```

**Review by kulotsystems — CHANGES_REQUESTED** [review]

**Comment by kulotsystems** [comment]
Please also delete the following:
- `node_modules` folder
- `package-lock.json` file
- `package.json` file

**Review by kulotsystems — CHANGES_REQUESTED** [review]
Thanks @miksdumb!

Lastly, please also delete the following:
- `node_modules` folder
- `package-lock.json` file
- `package.json` file

Thank you!

**Pull request merged**